geometry_msgs/Pose pose
geometry_msgs/Pose pose_microsteps
string zaber_response